packages/job-manager/src/index.ts

```diff
@@ -8,6 +8,7 @@ import { setupJobManagerContainer } from './setup-job-manager-container';
 (async () => {
     const jobManagerEntryPoint = new JobManagerEntryPoint(setupJobManagerContainer());
     await jobManagerEntryPoint.start();
-})().catch(() => {
+})().catch(error => {
+    console.log('Exception thrwon in job manager: ', error);
```

Comment:
```suggestion
    console.log('Exception thrown in job manager: ', error);
```

same for below code as well